examples/FullExample/README.md

```diff
@@ -0,0 +1,152 @@
+          device_password,                 //Device password provided by WolkAbout IoT Platform device upon device creation
+          &client,                         //MQQT Client
+          hostname,                        //MQQT Server
+          portno,                          //Port to connect to
+          PROTOCOL_JSON_SINGLE,              //Protocol specified for device
+          actuator_refs,                   //Array of strings containing references of actuators that device possess
+          2);                              //Number of actuator references contained in actuator_references
+wolk_connect(&wolk);
+```
+
+**Adding sensor readings:**
+
+Sensor types can be numeric, boolean and string and can containt more than one data parameter.
+Every type has it's own function for adding value eg. numeric sensor has:
+
+```c
+wolk_add_numeric_sensor_reading(&wolk,  //Context 
+                                "T",    //Sensor Reference
+                                23.4,   //Sensor Value
+                                0);     //UTC time of sensor value acquisition
+```
+and the
+```c
+wolk_add_multi_value_numeric_sensor_reading(&wolk,          //Context
+                                            "ACL",          //Sensor reference
+                                            accl_readings,  //Sensor values
+                                            3,              //Number of sensor dimensions
+                                            0);             //UTC time of sensor value acquisition
+```
+
+**Publishing actuator statuses:**
+```c
+wolk_publish_actuator_status(&wolk, "SLIDER_REF");
+```
+This will invoke the `actuator_status_provider` to read the actuator status, and publish actuator status.
+
+Process receives actuation commands from the platform when wolk_process is called.
+```c
+wolk_process (wolk_ctx_t *ctx, uint32_t tick)
+```
+Actuation handler and actuator status provider functions must be implemented by the user.
+Example function is included in the full example.
+```c
+static void actuation_handler(const char* reference, const char* value)
+
+static actuator_status_t actuator_status_provider(const char* reference)
+```
+
+**Device Configuration**
+
+Process receives configuration commands from the platform when wolk_proceess is called.
+```c
+wolk_process (wolk_ctx_t *ctx, uint32_t tick);
+```
+Configuration handler and configuration provider functions must be implemented by the user.
+Example function is included in the full example.
+```c
+static void configuration_handler(char (*reference)[CONFIGURATION_REFERENCE_SIZE],
+                                  char (*value)[CONFIGURATION_VALUE_SIZE],
+                                  size_t num_configuration_items)
+
+static size_t configuration_provider(char (*reference)[CONFIGURATION_REFERENCE_SIZE],
+                                     char (*value)[CONFIGURATION_VALUE_SIZE],
+                                     size_t max_num_configuration_items)
+```
+
```

Comment:
List function call for publishing configuration, explain that all configuration options are sent as a whole